### ISSUE TITLE
Undefined variable `port`

### ISSUE THREAD

**Issue by wzulfikar** [spatie/laravel-tail#20]
The variable `$port` in line 86 becomes undefined. It was defined in line 80 of previous commit. 

https://github.com/spatie/laravel-tail/commit/0a95dc04cf8be3660bdc37be49b3f8915c34ba19#diff-7082e6dd885caec42dde339a3c05e822

**Comment by freekmurze** [comment]
Whoops, fixed in `1.4.1`

**Comment by wzulfikar** [comment]
@freekmurze Sorry I just realised that the `$portCommand` also causes error when `port` isn't defined in `config/tail.php`. However, what causes me to open this issue is actually the `$port` variable (in last line). _Hence, the error `Undefined variable $port` is still there._

![screen shot 2017-03-17 at 7 14 37 pm](https://cloud.githubusercontent.com/assets/7823011/24042058/4e35b28a-0b4b-11e7-8254-679a2b7c2100.jpg)

Previously it was defined as `$port = 22` and the value will change accordingly if there's `port` key in `config/tail.php`. 

Since we added this customise-ssh-port feature, displaying port that's being used as info might be helpful. Thus, the `info` method will use the `$port` variable again to display which port is being used.
